scripts/bulk-access-control.md

```diff
@@ -0,0 +1,101 @@
+# bulk-access-control script
+[Back to the list of all defined scripts](./index.md)
+
+<!-- TOC -->
+* [bulk-access-control script](#bulk-access-control-management-script)
+  * [What it does?](#what-it-does)
+  * [Who can use it?](#who-can-use-it)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion](#conclusion)
+* [What is an Access Condition?](#what-is-an-access-condition)
+<!-- TOC -->
+## What it does?
+
+This script performs bulk changes to the access conditions of the items and bitstreams. 
+The Bulk Access Control Management focus on Access Control rather than resource policies. The goal will be to add / replace access condition for a set of objects from a user friendly interface that will abstract from the complexity/internal rules of DSpace’s resource policies and bundles as much as possible.
+It takes a json file as input to identify the items and bitstreams target of the changes and the access conditions to append or set.
```

Comment:
This statement is no longer accurate now that the UUID list is moved to a param.  The JSON file doesn't identify the target... it only identifies the changes to the access conditions

Reply:
This statement is wrong, as the UUID list is now a param.  The JSON file no longer "identifies the items and bitstreams target".

So, this should say something like:  "It takes a JSON file which identifies the access conditions to apply to the target resource(s).  The target resource(s) are provided via the `--uuid` parameter."